ui/app/styles/core/buttons.scss

```diff
@@ -332,11 +326,11 @@ a.button.disabled {
   }
 }
 
+// TODO HDS adoption cleanup: audit styles with design and see what to keep/remove once buttons are fully HDS
 // Existing class on <Hds::Copy::Button> component, modifying to match existing UI Structure buttons
 .hds-copy-button {
-  font-weight: $font-weight-semibold;
-  box-shadow: $box-shadow-low;
-  border-radius: $radius;
+  font-weight: $font-weight-semibold; // TODO delete
```

Comment:
just out of curiosity - what's the default font-weight for the copy button?

Reply:
There's no override option, it's only ever the clipboard icon 😄  [[docs](https://helios.hashicorp.design/components/copy/button?tab=code#icon-only)]
<img width="522" alt="Screenshot 2023-10-06 at 1 59 05 PM" src="https://github.com/hashicorp/vault/assets/68122737/c1d42702-0f45-4c54-8655-ab374e2e6f0d">

Reply:
If you consult the docs - the font weights for buttons appear to be `-token-typography-font-weight-regular: 400;`